recipes/dmd/meta.yaml

```diff
@@ -0,0 +1,25 @@
+package:
+  name: dmd
+  version: "2.065.0"
```

Comment:
Take a look at our example recipe and use a jijna variable for the version. It helps maintainers to update it later on.